src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -139,3 +183,33 @@ def _get_value(item, *args):
         return str(item) if item else ' '
     except (KeyError, TypeError, IndexError):
         return ' '
+
+
+def _get_build_trigger(image_update_trigger, git_commit_trigger):
+    if git_commit_trigger.strip():
+        return 'Git Commit'
+    if image_update_trigger.strip():
+        return 'Image Update'
+    return 'Manual'
+
+
+def _format_datetime(date_string):
+    from dateutil.parser import parse
+    try:
+        return parse(date_string).strftime("%Y-%m-%dT%H:%M:%SZ")
+    except ValueError:
+        logger.debug("Unable to parse date_string '%s'", date_string)
+        return date_string or ' '
+
+
+def _get_duration(start_time, finish_time):
+    from dateutil.parser import parse
+    try:
+        duration = parse(finish_time) - parse(start_time)
+        hours = "{0:02d}".format((24 * duration.days) + (duration.seconds // 3600))
+        minutes = "{0:02d}".format((duration.seconds % 3600) // 60)
+        seconds = "{0:02d}".format(duration.seconds % 60)
+        return "{0}:{1}:{2}".format(hours, minutes, seconds)
+    except ValueError:
+        logger.debug("Unable to get duration with start_time '%s' and finish_time '%s'", start_time, finish_time)
+        return ' '
```

Comment:
Same here--if the user inputs an invalid format, then failing silently and having unexpected behavior is probably not better than throwing an error.

Reply:
This is the same reason as above handling response datetime. We don't format user inputs here. In this case, `finish_time` could be empty.

Reply:
I didn't notice this was for table format.  Displaying blank is completely appropriate.